flow-typed/platform.js

```diff
@@ -16,6 +16,24 @@ declare type MWPState = {
 		enableServiceWorker: boolean,
 		requestLanguage: string,
 		supportedLangs: Array<string>,
+		member: { 
```

Comment:
Probably good to make this an intersection of exact types, with one of the 'constituent types' being `{| id: 0 |}`